photon-client/src/components/common/pv-slider.vue

```diff
@@ -25,9 +25,23 @@ const emit = defineEmits<{
   (e: "input", value: number): void;
 }>();
 
+// Debounce function
+function debounce(func: (...args: any[]) => void, wait: number) {
+  let timeout: ReturnType<typeof setTimeout>;
```

Comment:
what is this initialized to? not sure what clearTimeout(timeout); does for something that's uninitialized?